README.md

```diff
@@ -199,6 +199,18 @@ $ git status
 
     ```$ git show HEAD~1```
 
+4. A really nice colorized tree view: 
+
+    ```$ git log --graph --pretty=format:'%C(yellow)%h%Creset -%C(red)%d%Creset %s %Cgreen(%cd) %C(bold blue)<%an>%Creset' --abbrev-commit --date=local```
+
+5. This is a very long command so let's add this as an alias 'l', by doing:
+
+    ```$ git config --global --add alias.l "log --graph --pretty=format:'%C(yellow)%h%Creset -%C(red)%d%Creset %s %Cgreen(%cd) %C(bold blue)<%an>%Creset' --abbrev-commit --date=local"```
```

Comment:
Perhaps the alias should be named something else than just 'l'. E.g. tree-log?

As this is intended to be a tutorial and as easy to understand as possible